app/exec/extension/_lib/vsix-manifest-builder.ts

```diff
@@ -267,6 +267,30 @@ export class VsixManifestBuilder extends ManifestBuilder {
 					});
 				}
 				break;
+			case "repository":
+				if (_.isObject(value)) {					
+					let type = null, url = null;
+					Object.keys(value).forEach((key) => {

```

Comment:
Why are you looping through keys when all you expect to see is type and url? I'd use something like this.

    const {type, url} = value;
    if (!type) {
        throw new Error("Repository must have a 'type' property.");
    }
    if (type !== "git") {
        throw new Error("Currently 'git' is the only supported repository type.");
    }
    if (!url) {
        throw new Error("Repository must contain a 'url' property.";
    }
    this.addProperty("Microsoft.VisualStudio.Services.Links.Github", url);